src/services/CommentService.js

```diff
@@ -10,7 +9,7 @@ class CommentService {
   /**
  * users or managers can comment on a travel request
  * @static
- * @description POST /api/users/:tripRequestId/comment
+ * @description DELETE/api/trips/delete/comment
```

Comment:
I'm afraid that you used the wrong description because it should describe create comment instead of deleting comment.
i.e: 
```js
 * @description POST /api/users/:tripRequestId/comment
``